kyuubi-common/src/main/scala/org/apache/kyuubi/package.scala

```diff
@@ -51,11 +51,15 @@ package object kyuubi {
     val hadoop_version: String = props.getProperty("kyuubi_hadoop_version", unknown)
     val flink_version: String = props.getProperty("kyuubi_flink_version", unknown)
     val trino_version: String = props.getProperty("kyuubi_trino_version", unknown)
-    val branch: String = props.getProperty("branch", unknown)
-    val revision: String = props.getProperty("revision", unknown)
-    val revisionTime: String = props.getProperty("revision_time", unknown)
+    val branch: String = Option(props.getProperty("branch", unknown))
+      .filterNot(_.isEmpty).getOrElse(unknown)
```

Comment:
let extract a method and guard all properties

Reply:
@pan3793 good idea,i have updated, please check.